origin_ui/src/app/(dashboard)/Sidebar.tsx

```diff
@@ -17,25 +17,44 @@
  ***************************************************************/
 
 import Image from 'next/image'
-import styles from "../../app/page.module.css"
-import {Poppins} from "next/font/google";
+import Link from 'next/link'
+import {Typography, Box} from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import HomeIcon from '@mui/icons-material/Home';
+import BuildIcon from '@mui/icons-material/Build';
 
+import styles from "../../app/page.module.css"
 import PelicanLogo from "../../public/static/images/PelicanPlatformLogo_Icon.png"
 import GithubIcon from "../../public/static/images/github-mark.png"
-import {Typography, Box} from "@mui/material";
 
 export const Sidebar = () => {
 
     return (
         <Box>
-            <div className={styles.header} style={{display: "flex", flexDirection: "column", justifyContent:"space-between", padding:"1rem", top:0, position:"relative", zIndex:"1", overflow: "hidden", height: "100vh"}}>
-                <div style={{display:"flex"}}>
-                    <Image
-                        src={PelicanLogo}
-                        alt={"Pelican Logo"}
-                        width={32}
-                        height={32}
-                    />
+            <div className={styles.header} style={{display: "flex", flexDirection: "column", justifyContent:"space-between", padding:"1rem", top:0, position:"fixed", zIndex:"1", overflow: "hidden", height: "100vh"}}>
+                <div style={{display:"flex", flexDirection: "column"}}>
+                    <Link href={"/"}>
+                        <Image
+                            src={PelicanLogo}
+                            alt={"Pelican Logo"}
+                            width={36}
+                            height={36}
+                        />
+                    </Link>
+                    <Box pt={3}>
+                        <Link href={"/"}>
+                            <IconButton>
+                                <HomeIcon/>
+                            </IconButton>
+                        </Link>
+                    </Box>
+                    <Box pt={1}>
+                        <Link href={"/config/"}>
+                            <IconButton>
+                                <BuildIcon/>
+                            </IconButton>
+                        </Link>
+                    </Box>
                 </div>
                 <div>
```

Comment:
It seems that GitHub icon is not aligned with others in the column. Not sure if it's an existing bug or not.

Ref:
<img width="156" alt="Screenshot 2023-11-02 at 9 28 17 AM" src="https://github.com/PelicanPlatform/pelican/assets/41393704/6caba96e-1bbf-43aa-8bd9-87f453123120">